docs/BreakingChanges/059 Log files created by ObjectContext.CreateDatabase.md

```diff
@@ -0,0 +1,22 @@
+## 59: Log files created by the ObjectContext.CreateDatabase method
```

Comment:
{
  "Id": "59",
  "Title": "Log files created by the ObjectContext.CreateDatabase method",
  "Details": "When the CreateDatabase method is called either directly or by using Code First with the SqlClient provider and an AttachDBFilename value in the connection string, it creates a log file named filename_log.ldf instead of filename.ldf (where filename is the name of the file specified by the AttachDBFilename value). This change improves debugging by providing a log file named according to SQL Server specifications.",
  "Suggestion": "If the log file name is important for an app, the app should be updated to expect the standard _log.ldf file name format.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#sql",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.Objects.ObjectContext.CreateDatabase"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Data.Objects.ObjectContext.CreateDatabase